application/templates/_shared/_page_title.html

```diff
@@ -0,0 +1,3 @@
+{% macro page_title(title, error=False) %}
+{{ "Error: " if error }}{{ title }}
```

Comment:
Is there any reason not to include the `{% block page_title %}` wrapper in this macro, so it doesn't need duplicating in all of the other templates?

Reply:
@samuelhwilliams no good reason, other than that it _feels_ a bit weird to populate a block from inside a macro, for some reason...  Would reduce the amount of duplicate code on the page templates though, so maybe that's a reason to do it anyway?

Reply:
I don't think I mind the inclusion of `{% block page_title %} in each of the templates.  It's a bit of extra code, but I think it makes it clear to anyone working with the template what the structure of the page is.  I guess it would be OK to just call the macro from anywhere in a template and have the `<h1>` magically appear in the right place, but I'm suspicious of that kind of magic.

Reply:
Yeah I think it's probably more sensible to keep the blocks in the templates on second thought, too.